lib/_http_outgoing.js

```diff
@@ -67,6 +69,8 @@ const { CRLF, debug } = common;
 
 const kCorked = Symbol('corked');
 
+function nop() {}
```

Comment:
super minor nit... typically we use `noop` instead of `nop`

Reply:
`noop` and `nop` are used just about equally... 😄 In streams it's `nop`.